### DISCUSSION TITLE
Reporting

### DISCUSSION THREAD

**Discussion by gansky-alexander** [twentyhq/twenty#9183] [Ideas]
Add possibility to generate reports in the project.
For example usual report may be:
- funnel analysis of leads
- analise of time how much lead was in what state
- analize what is the most usual reasons we lost a projects 

Probably for this i need to implement something, it is okay.

**Comment by BOHEUS** [comment]
@gansky-alexander now it's possible with dashboards or soon you can write an app which will do it for you once SDK is stable and v1.0 is released (most likely this quarter)
